src/test/java/com/exasol/utils/JsonHelperTest.java

```diff
@@ -1,20 +1,20 @@
 package com.exasol.utils;
 
+import static org.hamcrest.CoreMatchers.equalTo;
```

Comment:
equalTo should exist in package `Matchers` too.

Reply:
changed